src/tools/r2rdump/UnwindInfo.cs

```diff
@@ -0,0 +1,70 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace R2RDump
+{
+    struct UnwindCode
+    {
+        public byte CodeOffset { get; }
+        public byte UnwindOp { get; } //4 bits
+        public byte OpInfo { get; } //4 bits
+
+        public byte OffsetLow { get; }
+        public byte OffsetHigh { get; } //4 bits
+
+        public ushort FrameOffset { get; }
+
+        public UnwindCode(byte[] image, ref int offset)
+        {
+            int off = offset;
```

Comment:
This doesn't look correct to me...is `offset` incremented correctly when it returns? It seems like `offset` should be used the entire time instead of initializing a new local variable.

Reply:
The UnwindCode (coreclr\src\inc\win64unwind.h) is a union, so it could be a 4 byte struct or an int. My program reads the same data twice, first as the fields in the struct, then as the int, so it doesn't increment offset on the first read.

Reply:
Ah, I didn't realize the native structure was a union.

@nattress how would you recommend reading in the bytes of this data type? Could we read the entire thing in as an unsigned integer and then parse the different bit ranges after that?

Reply:
Yeah I would just read the bytes directly and interpret them both ways. C# has union-like functionality with explicit layout support on structs but since we're just reading, we can parse into two fields, one of each type.

Reply:
Ok. Then I think this section looks good Amy 👍